server/tests/common.integration.js

```diff
@@ -3,6 +3,6 @@ import config from '../../config/config';
 
 const version = p.version.split('.').shift();
```

Comment:
Tests would not run, maybe due to this new code? `throw new Error('Config validation error: ' + error.message);`

Reply:
Hmm probably a .env issue, I'll look at my env and send you what i'm using that should work